database/pgx/pgx.go

```diff
@@ -209,7 +225,7 @@ func (p *Postgres) Lock() error {
 		return database.ErrLocked
 	}
 
-	aid, err := database.GenerateAdvisoryLockId(p.config.DatabaseName, p.config.SchemaName)
+	aid, err := database.GenerateAdvisoryLockId(p.config.DatabaseName, p.config.migrationsSchemaName, p.config.migrationsTableName)
```

Comment:
Note to self: document this backwards incompatible change
e.g. when updating to this version of `migrate`, you need to ensure that no migrations are currently running and no old versions of `migrate` will run at the same time as new versions.

Reply:
> Note to self: document this backwards incompatible change

@dhui **suggestion:** what do you think to create un GitHub issue named « v4.15.0 release » and add this note in todo list in this issue?

Reply:
No worries, I've create a draft release and add this note there